human_path_prediction/HumanPose[]  goals
---
bool                         success
string                       message
